index.js

```diff
@@ -143,88 +143,85 @@ module.exports = function (urlRoot, pathRoot) {
 
     // if requested url is in our interest
     if (!req.baseUrl && req.url.indexOf('/' + baseUrl) !== 0) {
-      return next();
-    }
+      next();
+    } else {
```

Comment:
Is this a good idea?

Reply:
Maybe this not a good review comment. I mean, I don't why you didn't want to use early return here but this function is already a big, complex, hard do read function. One more indentation for nearly 100 lines of code doesn't look like a good thing.

Reply:
Oh. Yeah.

I search the way to rewrite my default responses

This is my code where I tried to investigate how can I just rewrite my default response to `/api/v1/list_crops` by a definite state

```javascript
var express = require('express');
var apiMocker = require('connect-api-mocker');

var app = express();

const { state, basePath = '/api/v1' } = process.env;

if (!state) {
  console.log('\n');
  console.log('\x1b[31m%s\x1b[0m', 'Define "state" (state="/path/to/json" nodemon)');
  console.log('\n');

  return;
}

console.log('\n', 'State:', `"${state}"`, '\n');
console.log('\n', 'basePath:', `"${basePath}"`, '\n');

app.use(function(req, res, next) {
  res.header("Access-Control-Allow-Credentials", true);
  res.header("Access-Control-Allow-Origin", req.headers.origin);
  res.header("Access-Control-Allow-Headers", "authorization,timezone,content-type");
  next();
});

// default responses
app.use(`${basePath}/list_crops`, apiMocker('states/_base/ru/api/v1/list_crops'));
...
// definite state, where default response can be changed
app.use(`${basePath}`, apiMocker(state));

```

And then I thought what If I just return `next();` somehow and then return the response at the end of my code.

I will rollback these changes because I could not find solution yet

But anyway, it would be a nice feature to rewrite default responses. It would be `killer` feature for me ;)

Reply:
Oh
I fixed some code issues with `eslint` tool and removed some annoying warnings from `eslint` rules

```javascript
"max-len": "off",
"func-names": "off",
"no-useless-escape": "off",
"global-require": "off",
"import/no-dynamic-require": "off",
"consistent-return": "off",
"prefer-destructuring": "off",
"camelcase": "off",
"comma-dangle": ["error", "never"]
```